frontend/src/app/main/data/users.cljs

```diff
@@ -158,7 +162,7 @@
     (watch [_ state stream]
       (let [{:keys [id] :as profile} (:profile state)]
         (->> (rp/mutation :update-profile-props {:props {:onboarding-viewed true}})
-             (rx/map (constantly fetch-profile)))))))
+             (rx/map (constantly (fetch-profile))))))))
```

Comment:
you can replace all these `(constantly (fetch-profile))` for just `fetch-profile`. This line will be...
```clojure
(rx/map fetch-profile)
```

Reply:
It seems not. I need a function that returns a function. If I change this line as you said, I get an "Invalid arity: 0" error.